Gemfile.lock

```diff
@@ -0,0 +1,206 @@
+GEM
```

Comment:
what happens if i delete this file and then run the server?

Reply:
The server will still run normally as ```Gemfile.lock``` is only a snapshot of the application, it contains the exact version of gems being used in our application